python/rgbmatrix/core.pyx

```diff
@@ -83,18 +83,100 @@ cdef class FrameCanvas(Canvas):
         def __get__(self): return (<cppinc.FrameCanvas*>self.__getCanvas()).pwmbits()
         def __set__(self, pwmBits): (<cppinc.FrameCanvas*>self.__getCanvas()).SetPWMBits(pwmBits)
 
+cdef class RGBMatrixOptions:
+    def __cinit__(self):
```

Comment:
The problem is how to split the args between Python and the C++ lib. Because sample subclasses can (and do) add additional command line args. 

Theoretically, it is possible to tell Python's arg parser to split the args into the "known" args (those added to Python's arg parser) and additional ones that were not defined (which will be passed to C++). I am not sure how well it works. Some code needs to handle the "-h/--help" switch to print the command line. I need to see if this is possible to split those between Python's parser and C++ and it will be inconsistent.

I really think this is a very minor issue since most users will only use the demo briefly before writing their own code that uses the classes directly. 

And in any case, the situation in this pull request is not worse than before, so I don't think this should hold the pull requests.

I will look at how/if this can be done later.